libs/plugins/stream-client/src/lib/simulator/simulator.component.html

```diff
@@ -1,6 +1,11 @@
 <div *ngIf="simulateActivity" class="activity column">
   <div class="activity__details">
-    <img class="activity__icon" [src]="iconUrl" alt="Activity icon" />
+    <flogo-activity-icon
+        [iconUrl]="iconUrl"
+        [customAltText]="'Activity icon'"
+        [isTerminal]="isTerminal"
```

Comment:
isTerminal is an invalid reference as the property doesn't exist on the simulator component class. 

It can be removed as it's an optional input.